Личный проект: по тропе Аппалачей

arloktev opened the pull request:
---
:mortar_board: [Личный проект: по тропе Аппалачей](https://up.htmlacademy.ru/nodejs/3/user/129863/tasks/14)

arloktev committed:
feat(express): добавил сервер на express, обработку необходимых роутов

dean992008 commented on package.json:
```diff
@@ -6,7 +6,10 @@
+    "start": "node ./src/service/service.js",
+    "server": "node ./src/service/service.js --server 3000",
+    "server:dev": "nodemon ./src/service/service.js --server 3000",
+    "start-front-server": "nodemon ./src/express/express.js"
```
Порекомендую с `start-front-server` сделать как и с `server`. `start-front-server` запускать через node, а `start-front-server:dev` через `nodemon`

dean992008 commented on src/express/express.js:
```diff
@@ -0,0 +1,17 @@
+
+const app = express();
+
+app.use(`/`, mainRoutes);
```
Я бы реокмендовал вынести роутинг в отсдельный модуль, по такому же принципу как мы делаем в `articles-routes.js` и использовать его здесь: 
```
app.use(router);
```

dean992008 commented on src/express/routes/articles-routes.js:
```diff
@@ -0,0 +1,12 @@
+
+const articlesRoutes = new Router();
+
+articlesRoutes.get(`/category/:id`, (req, res) => res.send(`/articles/category/:id`));
```
Нужно сделать чтобы `http://localhost:8080/articles/category/3` возвращая строку `/articles/category/3`

arloktev committed:
правки после кодревью
refactor(router): вынес роутер в отдельный модуль
fix(articlesRoutes): скорректировал вывод динамического параметра
feat(scripts): добавил запуск express в двух режимах
